yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
good point that you removed the `package-lock` from the root which didn't belong there
but this one we probably don't need it in the root either do we?

Reply:
removed